.gitignore

```diff
@@ -3,5 +3,6 @@ hosts/
 !hosts/localhost
 roles/
 inventory
-files/jobs/
-files/nfv_jobs_config/
+files/
+!files/filebeat.yml
```

Comment:
I changed this so that the git clones of whatever repository, be that private, public, etc don't result in an untracked change.

Need to be careful if we decide to add any new files here.